index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title>Portfolio-Mobile</title>
+</head>
+
+<body>
+    <header>
+        <a href="#" class="name">Victor</a>
+        <nav>
+            <img src="pictures/Enabled.png" alt="menu">
+        </nav>
+    </header>
```

Comment:
- [ ] Your navigation bar looks little different from original. You can see it has little space on top, right and left side. Please fix these.

**Original:**
![image](https://user-images.githubusercontent.com/38635158/124994236-38cd9480-e04e-11eb-9b13-76bb735d118e.png)
___
**Yours:**
![image](https://user-images.githubusercontent.com/38635158/124994481-96fa7780-e04e-11eb-84d4-bbbd84041644.png)